policy/microos/rancher.te

```diff
@@ -0,0 +1,87 @@
+policy_module(rancher, 1.0.0)
+
+gen_require(`
+    type container_runtime_t, unconfined_service_t;
+')
+
+########################
+# type rke_logreader_t #
+########################
+gen_require(`
+        type container_runtime_t, unconfined_service_t;
+        type container_log_t;
+        type syslogd_var_run_t;
+        type var_log_t;
+        class dir { read search };
+        class file { open read };
+        class lnk_file { getattr read };
+')
+container_domain_template(rke_logreader)
+virt_sandbox_domain(rke_logreader_t)
+corenet_unconfined(rke_logreader_t)
+allow rke_logreader_t container_log_t:dir { open read search };
+allow rke_logreader_t container_log_t:lnk_file { getattr read };
+allow rke_logreader_t container_log_t:file { getattr open read };
+allow rke_logreader_t container_var_lib_t:dir search;
+allow rke_logreader_t container_var_lib_t:file { getattr open read };
+allow rke_logreader_t container_var_lib_t:lnk_file { getattr read };
+allow rke_logreader_t syslogd_var_run_t:dir read;
+allow rke_logreader_t syslogd_var_run_t:file { getattr map open read };
```

Comment:
> I just had a look at the parts in k3s-selinux and rke2-selinux and tried to find out which files differ between centos8 and microos.
> I then copied the files from centos8 and tried to adapt all paths and names and such.

I found no differences for CentOS8 and MicroOS in the files from rke2-selinux and k3s-selinux, so I was hoping that this might also be true for rancher-selinux.

Unfortunately I am lacking the knowledge to verify that. I was just trying to put together the pieces to help getting this going...

Reply:
I tested this and it looks like map is not needed on microos
```suggestion
allow rke_logreader_t syslogd_var_run_t:file { getattr open read };
```

Reply:
Fixed.